### DISCUSSION TITLE
Web3Model in React Native - the wallet service providers such as Metamask do not appear (Possible unhandled promise rejection (id:2); SyntaxError: JSON Paerse error: Unexpected character: B

### DISCUSSION THREAD

**Discussion by AmineBenzaazaa** [WalletConnect/walletconnect-monorepo#4469] [Developer Support]
React Native - Connect wallet doesn't show Wallet service providers (Possible unhandled promise rejection (id:2); SyntaxError: JSON Paerse error: Unexpected character: B

When I try to click on the "Connect" button, a popup modal appears. However, when I select "All Wallets", the wallet service providers such as Metamask do not appear. Instead, I receive an error message that says "Possible unhandled promise rejection (id:2); SyntaxError: JSON Parse error: Unexpected character: B."
![WhatsApp Image 2024-04-22 at 12 04 09](https://github.com/WalletConnect/walletconnect-monorepo/assets/61782040/02dcc7c5-1dbc-41b7-bdfa-e4a19d4bf837)
![WhatsApp Image 2024-04-22 at 12 04 20](https://github.com/WalletConnect/walletconnect-monorepo/assets/61782040/a9607338-cacb-46f6-b0ab-8cfb66e4db23)
![WhatsApp Image 2024-04-22 at 12 04 44](https://github.com/WalletConnect/walletconnect-monorepo/assets/61782040/70384d88-1a2c-4f3a-9278-fa58ae7dd1d7)

**Comment by glitch-txs** [comment]
hi, could you open a GitHub issue in the React Native repository?
